### PR TITLE
Clarify use of relevant [admins] section

### PR DESCRIPTION
**Pull request by ahayes**
## Overview

Helps to avoid problems associated with apache/couchdb#777.

Please check that I have referenced the :config section properly. I didn't know what I was doing and had no easy way to test.

## Checklist

- [ ] Documentation is written and is accurate;
- [ ] `make check` passes with no errors
- [ ] Update [rebar.config.script](https://github.com/apache/couchdb/blob/master/rebar.config.script) with the commit hash once this PR is rebased and merged

### CODE REVIEW

File: src/config/auth.rst [hunk 1]
```diff
@@ -29,9 +29,11 @@ Server Administrators
     users. This configuration is known as `Admin Party`, and is not recommended
     for in-production usage. You can crash the party simply by creating the
     first admin account. CouchDB server administrators and passwords are not
-    stored in the ``_users`` database, but in the ``local.ini`` file, which
-    should be appropriately secured and readable only by system
-    administrators::
+    stored in the ``_users`` database, but in the last ``[admins]`` section
+    that CouchDB finds when loading. See :config:intro
```
[the remark this answers is not in the document]
Comment (flimzy) [reply src/config/auth.rst]: Would it make sense to say "in the last [admins] section of the ini file" or similar, to make it more explicit where this admins section should be found?

[the remark this answers is not in the document]
Comment (wohali) [reply src/config/auth.rst]: No, that's not quite right, because it's "the last [admins] section of the last ini file in the chain"

**Comment by wohali** [comment]
@flimzy that's covered here: https://github.com/apache/couchdb-documentation/blob/master/src/config/intro.rst#introduction-to-configuring

to which @ahayes added a link with this PR, which is a Very Good Thing :D

**Comment by wohali** [comment]
@ahayes we'd love to merge this but there are a few minor changes requested, can you help? Thanks!

**Comment by wohali** [comment]
@ahayes I'd love to close this out and get this merged in, but your PR needs a few small changes. Would you have time to get these cleaned up soon? If not, I'm afraid I'll have to close this PR without merging it. :(

Thanks again for your contributions, they are most welcome!!

**Comment by ahayes** [comment]
I'm a n00b! I need to figure out how to merge. Am happy to. Give me an hour.

**Comment by ahayes** [comment]
I think I have managed to resolve all your suggestions and make TravisCI happy too. (I have filed a feature request with GitHub to display cursor position in their text editor so small fixes on the web are easier!)

**Comment by wohali** [comment]
Perfect @ahayes  - thanks again for your contribution!

**Pull request merged**